HighPerformanceComputing.ctv

```diff
@@ -108,7 +108,7 @@
         via MPI profiling libraries, such as fpmpi, mpiP, or TAU.
       </li>
       <li>An alternative is provided by the <pkg>nws</pkg> (NetWorkSpaces)
-        packages from REvolution Computing. It is the successor to the
+        packages from Revolution Computing. It is the successor to the
```

Comment:
That is how they spelled it in the, see eg https://techcrunch.com/2009/10/20/revolution-computing-raises-9-million/: